templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -59,8 +59,8 @@ fi
 echo "machine pulp
 login admin
 password password
-" | cmd_stdin_prefix bash -c "cat > /root/.netrc"
-cmd_stdin_prefix bash -c "chmod og-rw /root/.netrc"
+" | cmd_stdin_prefix bash -c "cat > ~pulp/.netrc"
+cmd_stdin_prefix bash -c "chmod og-rw ~pulp/.netrc"
```

Comment:
This is because we run the tests as the same user pulp?

Reply:
Yes